File: Practice Solutions/9-12-19/ownWay.py
tests = int(input())
for test in range(0,tests):

    N = int(input()) 

    inString = input()
    
    print("Case #{}: ".format(test + 1), end="")

    for s in inString:
        if s == 'S':
            print('E', end="")
        else:
            print('S', end="")
    print()

File: Practice Solutions/9-12-19/ownWay.cpp
#include<bits/stdc++.h>
using namespace std;

#define x first
#define y second

typedef long double ld;

typedef unsigned long long lli;


int maxOf(int a, int b){
    return (a > b ? a : b);
}

int minOf(int a, int b){
    return (a < b ? a : b);
}

int main(){
    cin.tie(NULL);
    ios_base::sync_with_stdio(false);

    int tests = 0;
    cin >> tests;
    for(int test = 0; test < tests; test++){

        int N;
        cin >> N;

        char inChar;
        cout << "Case #" << test + 1 << ": ";

        for(long i = 0; i < 2*N - 2 ; i++){
            cin >> inChar;
            cout << (inChar == 'E'? 'S' : 'E');
        }

        cout << endl;
    }
}

// https://codingcompetitions.withgoogle.com/codejam/round/0000000000051705/00000000000881da